index.html

```diff
@@ -249,15 +255,30 @@ <h3>Calculates the Pressure conversion between two specified units.</h3>
                 </a>
             </div>
         </div>
+        <div class="box">
+            <div class="content">
+                <h2>4 Band Resistance Calculator</h2>
+                <h3>Calculates the theoritical resistance value of a Resistor having 4 Colour Bands.</h3>
+                <a href="./Calculators/4_Band_Resistance_Calculator/index.html" target="_blank">
```

Comment:
Pls change the `underscore` to `hyphen` here @amansgith

Reply:
Done